package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "express-rescue",
-  "version": "1.1.31",
+  "version": "1.1.33",
```

Comment:
Since this PR doesn't change the lib's code, we don't need to bump the version yet :)

Reply:
Oh yeah, that makes sense! Done!